compiler/luci/pass/src/FusePreActivationBatchNormPass.cpp

```diff
@@ -456,6 +463,11 @@ bool fuse_mul_with_conv(luci::CircleMul *mul)
 
       // Update CONV weights
       update_conv_weights_with_gamma(conv, gamma);
+
+      // Update origin
+      // TODO need to remove const
+      luci::add_origin(const_cast<luci::CircleNode *>(static_cast<const luci::CircleNode *>(conv)),
```

Comment:
@jinevening I might know why you return conv with `const` keyword. But to add origin to `conv`, maybe `const` keyword should be removed. How about your opinion?

Reply:
Using `const_cast` looks good to me. I guess that the code can be simplified as below.

```suggestion
      luci::add_origin(const_cast<luci::CircleConv2D *>(conv)),
```